py/packages/genkit/src/genkit/veneer/veneer.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         else:
             for plugin in plugins:
                 if isinstance(plugin, Plugin):
-                    plugin.attach_to_veneer(veneer=self)
+                    plugin(self)
```

Comment:
this line may be misleading, as calling plugin seamlessly attaches it in veneer's registry